core/services/keeper/registry_interface.go

```diff
@@ -23,13 +25,16 @@ const (
 	RegistryVersion_1_2
 )
 
+const ActiveUpkeepIDBatchSize int64 = 10000
```

Comment:
did we do any testing to confirm that this batch size easily fits under the gas limit?

Reply:
yes. both akshay and i tested fetching 10K batch size and it works fine. 
i even tested 50k and it still works fine in kovan testnet but i figure it's just safer to use a lower limit for now.

Reply:
okay cool. 10k sounds good to me